process/views/api.py

```diff
@@ -142,6 +144,7 @@ def close_collection(request):
 def create_collection_file(request):
     if request.method == "POST":
         input = json.loads(request.body)
+        input["path"] = os.path.join(settings.FILES_STORE, input['path'])
```

Comment:
Use consistent quote characters.

```suggestion
        input["path"] = os.path.join(settings.FILES_STORE, input["path"])
```